tests/utils.py

```diff
@@ -78,6 +78,16 @@ def create_testdir(*extra_path):
     except OSError:
         pass
 
+def cleanup_testdir(*extra_path):
+    try:
+        path = TESTDIR_NAME
+        if extra_path:
+            path = os.path.join(TESTDIR_NAME, *extra_path)
+
+        subprocess.run(["/bin/bash", "-c", f"rm -rf '{path}'"])
```

Comment:
why not just 

```suggestion
        subprocess.run(["rm", "-rf", path])
```
or even better more portable

```suggestion
        shutil.rmtree(path, ignore_errors=True)
```
or alike?